nevergrad/functions/mixsimulator/core.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/Foloso/MixSimulator/tree/nevergrad_experiment
+
+import numpy as np
+from ..base import ExperimentFunction
+from mixsimulator.MixSimulator import MixSimulator  #type: ignore
```

Comment:
```suggestion
from mixsimulator.MixSimulator import MixSimulator  #type: ignore
from ..base import ExperimentFunction
```
(standard order: from more general to more specific, so external first and internal afterwards)